ui/pages/add-snap-account/snap-account-detail-page/header.tsx

```diff
@@ -0,0 +1,213 @@
+            variant={BUTTON_VARIANT.LINK}
+            marginRight={4}
+            onClick={() => history.back()}
+          >
+            {t('snapDetailsCreateASnapAccount')}
+          </Button>
+          <Icon name={IconName.ArrowRight} marginRight={4} />
+          <Text>{snapTitle}</Text>
+        </Box>
+        <Box
+          display={Display.Flex}
+          justifyContent={JustifyContent.spaceBetween}
+        >
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Row}
+            alignItems={AlignItems.center}
+          >
+            <Text variant={TextVariant.headingLg} marginRight={1}>
+              {snapTitle}
+            </Text>
+            {isInstalled && (
+              <Tag
+                label={t('snapDetailsInstalled') as string}
+                labelProps={{
+                  color: TextColor.textAlternative,
+                }}
+                className=""
+                height={2}
+              />
+            )}
+          </Box>
+          <Box>
+            {isInstalled && updateAvailable && (
+              <Button
+                variant={BUTTON_VARIANT.PRIMARY}
+                marginRight={1}
+                onClick={() => {
+                  setShowConfigPopoverType(ConfigureSnapPopupType.INSTALL);
+                  setShowConfigPopover(true);
+                }}
+              >
+                {t('snapUpdateAvailable')}
+              </Button>
+            )}
+            {isInstalled && (
+              <Button
+                variant={BUTTON_VARIANT.PRIMARY}
+                onClick={() => {
+                  setShowConfigPopoverType(ConfigureSnapPopupType.CONFIGURE);
+                  setShowConfigPopover(true);
+                }}
+              >
+                {t('snapConfigure')}
+              </Button>
+            )}
+            {!isInstalled && (
+              <Button
+                variant={BUTTON_VARIANT.PRIMARY}
+                onClick={() => {
+                  setShowConfigPopoverType(ConfigureSnapPopupType.INSTALL);
+                  setShowConfigPopover(true);
+                }}
+              >
+                {t('snapInstall')}
+              </Button>
+            )}
+          </Box>
+        </Box>
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Row}
+          alignItems={AlignItems.center}
+        >
+          <Box
+            display={Display.Flex}
+            justifyContent={JustifyContent.center}
+            alignItems={AlignItems.center}
+            style={{
+              borderRadius: '50%',
+              height: '32px',
+              width: '32px',
+            }}
+            borderWidth={1}
+            borderColor={BorderColor.borderMuted}
+            padding={[2, 2, 2, 2]}
+            marginRight={1}
+          >
+            <img src={iconUrl} className="snap-detail-icon" />
+          </Box>
+          {developer === 'Metamask' && (
```

Comment:
We should make the value `MetaMask` a constant in a separate file

Reply:
created a new constants file